README.md

```diff
@@ -372,22 +367,22 @@ was more awkward than it is now.
 Also, the video does not describe the user's key presses and button
 presses.
 
-But here are some general user-interface aspects that you should
-have gathered from the video.
+Here are some general user-interface aspects that you should
+have gathered from the video:
 
-Each node that the user creates is added to a surface (the fragment);
-this surface is displayed in map view in the Fragment View.  Of course, if
-the new nodes all form a line, only a line, rather than a surface,
-is shown in the Fragment view.  Once the user creates nodes
+* Each node that the user creates is added to a surface (the fragment);
+this surface is displayed in the map view under the Fragment View.  Of course, if
+the new nodes form a line, rather than a surface,
+it is shown in the Fragment View.  Once the user creates nodes
```

Comment:
```suggestion
* Each node that the user creates is added to a surface (the fragment);
a map view of this surface is displayed in the Fragment View window.
If all the nodes lie in a straight line,
then the Fragment View will display a line rather than a surface.  Once the user creates nodes
```
Sure. However, let's keep it short and to the point; too many commas make it hard to comprehend.
I have re-arranged it and dropped "Of course" as it adds unnecessary detail.

Reply:
Unresolving comment. Modifications were confirmed to be okay (see https://github.com/KhartesViewer/khartes/pull/17#issuecomment-1793367671)